general-concepts/manifest/text.xml

```diff
@@ -7,68 +7,16 @@
 <title>Generating the Manifest</title>
 <body>
 <p>
-In the tree, every package has a <c>Manifest</c> file. This file lives in the same
-directory as the ebuilds for the package. The <c>Manifest</c> file contains digests
-(currently RMD160, SHA1, SHA256, SHA512 and WHIRLPOOL) and file size data for every
+In the tree, every package has a <c>Manifest</c> file. This file lives
+in the same directory as the ebuilds for the package. The <c>Manifest</c> file
+contains digests (currently BLAKE2B and SHA512) and file size data for every
 file in the directory and any subdirectories. This is used to verify integrity.
-The <c>Manifest</c> may also be digitally signed.
 </p>
 
 <p>
-To generate the <c>Manifest</c>, use <c>ebuild foo.ebuild manifest</c>. When
-committing, the <c>Manifest</c> file must be regenerated to handle any
-changes <d/> <c>repoman</c> will do this automatically.
-</p>
-</body>
-</section>
-
-<section>
-<title>Signing the Manifest using your GPG key</title>
-<body>
-<p>
-Requirements:
-</p>
-
-<ul>
-  <li>&gt;=sys-apps/portage-2.0.51_pre10</li>
-  <li>&gt;=app-crypt/gnupg-1.2.4</li>
-</ul>
-
-<p>
-Key Setup:
-</p>
-
-<ul>
-  <li>
-    <uri link="https://www.gentoo.org/doc/en/gnupg-user.xml#doc_chap2">Create</uri>
-    a new DSA GnuPG key with at least a 1024 bit keylength, an expiration
-    period no longer than 6 months and a good passphrase.
-  </li>
-  <li>
-    <uri link="https://www.gentoo.org/doc/en/gnupg-user.xml#doc_chap3">Upload</uri>
-    the key to a keyserver.
-  </li>
-</ul>
-
-<p>
-Portage Configuration:
-</p>
-
-<ul>
-  <li>
-    Set <c>PORTAGE_GPG_DIR</c> to your <c>~/.gnupg/</c> directory
-    (or the directory where the keyring with your new key is).
-  </li>
-  <li>Set <c>PORTAGE_GPG_KEY</c> to the key id of your new key.</li>
-  <li>Set FEATURES="sign".</li>
-</ul>
```

Comment:
AFAICS, `PORTAGE_GPG_KEY` is not mentioned anywhere else. This section shouldn't be removed, but be updated for git signing instead.

Reply:
But what does git signing have to do with Manifests?

Reply:
Yeah, remove it for now. `ebuild-maintenance/git/` already says something about openpgp signing, it only doesn't mention `PORTAGE_GPG_KEY`.